lib/gruf/autoloaders.rb

```diff
@@ -51,26 +51,26 @@ def reload
       #
       # @return [::Gruf::Controllers::Autoloader]
       #
-      # rubocop:disable ThreadSafety/InstanceVariableInClassMethod
```

Comment:
This cop was renamed to `ThreadSafety/ClassInstanceVariable` in `rubocop-thread_safety` v0.6.0 released 2 months ago.

https://github.com/rubocop/rubocop-thread_safety/blob/master/CHANGELOG.md#060